client/components/tinymce/plugins/wpcom-view/views/simple-payments/index.jsx

```diff
@@ -19,15 +20,44 @@ import { getSelectedSiteId } from 'state/ui/selectors';
 import formatCurrency from 'lib/format-currency';
 import QuerySimplePayments from 'components/data/query-simple-payments';
 import QueryMedia from 'components/data/query-media';
+import QuerySitePlans from 'components/data/query-site-plans';
+import { hasFeature, getSitePlanSlug } from 'state/sites/plans/selectors';
+import { FEATURE_SIMPLE_PAYMENTS } from 'lib/plans/constants';
 
 class SimplePaymentsView extends Component {
 	render() {
-		const { translate, productId, product, siteId } = this.props;
+		const {
+			translate,
+			productId,
+			product,
+			siteId,
+			shouldQuerySitePlans,
+			planHasSimplePaymentsFeature,
+		} = this.props;
 
 		if ( ! product ) {
 			return <QuerySimplePayments siteId={ siteId } productId={ productId } />;
 		}
 
+		if ( shouldQuerySitePlans ) {
+			return <QuerySitePlans siteId={ siteId } />;
```

Comment:
We still need to query if `siteId` changes, let's remove the `if( shouldQuerySitePlans ) {`